test/benchmark.js

```diff
@@ -21,17 +21,37 @@ const isWin32 = require('os').platform() === 'win32';
 const npmScript = isWin32 ? 'npm.cmd' : 'npm';
 
 const testDir = resolve('.tmp-hexo-theme-unit-test');
+const zeroEksDir = process.env.TRAVIS_BUILD_DIR
+  ? join(process.env.TRAVIS_BUILD_DIR, '0x')
+  : resolve(testDir, '0x');
```

Comment:
To be compatible with current Travis CI configuration. Should be removed in the future.